community/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['127.0.0.1', 'localhost', 'testserver']
```

Comment:
Unrelated change?

Reply:
No, the build will fail because the syndication feed framework needs `testserver` to be allowed for `django_distill` builds.

Reply:
Okay 👍